app/assets/javascripts/channels/synchronized_editor_channel.js

```diff
@@ -23,13 +24,18 @@ $(document).on('turbolinks:load', function () {
 
         received(data) {
           // Called when there's incoming data on the websocket for this channel
-          if (current_user_id !== data['current_user_id']) {
-            CodeOceanEditor.applyChanges(data['delta']['data']);
+          if (data.command === 'editor_change' && current_user_id !== data['current_user_id']) {
```

Comment:
As discussed, we should not just rely on the user ID here, but also consider the user type. Further, we should switch to some generic UUID where possible, to prevent some synchronization issues with multiple tabs / windows.